docs/most-common-issues.md

```diff
@@ -0,0 +1,14 @@
+Before you do anything else try to remove the project again from your computer and clone it again
```

Comment:
This should only be mentioned as a last resort.
I think it'd be a good idea to tell the user to check the yarn version and point that there are issues with yarn !== 0.28.4